mslib/msui/ui/ui_mscolab_connect_dialog.ui

```diff
@@ -303,28 +304,27 @@
            </property>
           </widget>
          </item>
-         <item row="1" column="0">
+         <item row="0" column="0">
           <widget class="QLabel" name="httpPasswordLabel">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
-         <item row="0" column="1">
-          <widget class="QLineEdit" name="httpUsernameLe">
-           <property name="placeholderText">
-            <string>Server Auth Username</string>
```

Comment:
I would prefer to have an auth user name too. When someone has it in his json file we may not need to show it on the ui. But on serverside we should have it and it is not the same word.

Reply:
There is a username on the server side, but on the client it is fixed to "mscolab".
For authentification, we do not need a username. 
Why needlessly complicate the user interface?

I simply added a new "mscolab" user with the same password hash to the mss-stage config file.

Reply:
ok, on client site only a string is usally meant as API key, we could also think on this approach. On server site I would prefer more than one than too.  Currently we would have only for msui, but when there is another client/service I whish to give to that another key.

Compared to BBB there I have to use the same API key for the wiki, nc, greenlight, RC, MM. I dislike this.


By using the HTTPBasicAuth the https://flask-httpauth.readthedocs.io/en/latest/ as we do (see also wms.py) we can shift the definitions to the web server.  This is also a good argument to keep the behaviour of HTTPBasicAuth.
e.g. if we for example move the socketio Server to a different location path, we could protect the whole location path by the webservers auth definition.  https://github.com/miguelgrinberg/Flask-SocketIO/issues/302